.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules
 npm-debug.log
 docs
+notes
```

Comment:
The notes folder is part of my workflow during development where I jot down todos, notes on the project etc. I obviously don’t want to commit that though so I add it to the `.gitignore`. I can move this out of my local project root to keep our gitignore clean however.